Source/Core/Core/IOS/DolphinDevice.cpp

```diff
@@ -26,15 +26,16 @@ namespace
 {
 enum
 {
-  IOCTL_DOLPHIN_GET_SYSTEM_TIME = 0x01,
+  IOCTL_DOLPHIN_GET_ELAPSED_TIME = 0x01,
   IOCTL_DOLPHIN_GET_VERSION = 0x02,
   IOCTL_DOLPHIN_GET_SPEED_LIMIT = 0x03,
   IOCTL_DOLPHIN_SET_SPEED_LIMIT = 0x04,
   IOCTL_DOLPHIN_GET_CPU_SPEED = 0x05,
   IOCTL_DOLPHIN_GET_REAL_PRODUCTCODE = 0x06,
   IOCTL_DOLPHIN_DISCORD_SET_CLIENT = 0x07,
   IOCTL_DOLPHIN_DISCORD_SET_PRESENCE = 0x08,
-  IOCTL_DOLPHIN_DISCORD_RESET = 0x09
+  IOCTL_DOLPHIN_DISCORD_RESET = 0x09,
+  IOCTL_DOLPHIN_GET_SYSTEM_TIME = 0x0A
```

Comment:
Please add an extra comma to prevent diff noise in future additions.

Reply:
Resolved in https://github.com/dolphin-emu/dolphin/pull/11269/commits/37888687c569080d4fa66928273de84b4ae01aec.